api/seeds/development/apds.js

```diff
@@ -67,26 +67,6 @@ exports.seed = async knex => {
     .returning('id');
 
   await knex('apd_incentive_payments').insert([
-    {
-      apd_id,
-      year: '2018',
-      q1_eh_count: 0,
-      q2_eh_count: 0,
-      q3_eh_count: 0,
-      q4_eh_count: 0,
-      q1_eh_payment: 0,
-      q2_eh_payment: 0,
-      q3_eh_payment: 0,
-      q4_eh_payment: 0,
-      q1_ep_count: 0,
-      q2_ep_count: 0,
-      q3_ep_count: 0,
-      q4_ep_count: 0,
-      q1_ep_payment: 0,
-      q2_ep_payment: 0,
-      q3_ep_payment: 0,
-      q4_ep_payment: 0
-    },
```

Comment:
It hasn't been FFY 2018 for a while so there's no good reason to keep this data in here... 😛